website/docs/install-config/configuration/configuration.mdx

```diff
@@ -65,38 +65,69 @@ To check if your config has been applied correctly, you can run the following co
+- `AUTHENTIK_POSTGRESQL__USER`: Username that authentik will use to authenticate with PostgreSQL.
+- `AUTHENTIK_POSTGRESQL__PASSWORD`: Password for PostgreSQL authentication. If not set, it defaults to the value of the `POSTGRES_PASSWORD` environment variable (this fallback is specific to the default Docker Compose setup).
+- `AUTHENTIK_POSTGRESQL__NAME`: The name of the database for authentik to use.
+
+:::info Hot-reloading
+The `AUTHENTIK_POSTGRESQL__HOST`, `AUTHENTIK_POSTGRESQL__PORT`, `AUTHENTIK_POSTGRESQL__USER`, and `AUTHENTIK_POSTGRESQL__PASSWORD` settings support hot-reloading and can be changed without restarting authentik. However, adding or removing read replicas requires a restart.
+:::
+
+### SSL/TLS Settings
+
+Configure SSL/TLS to secure the connection to your PostgreSQL server.
+
+- `AUTHENTIK_POSTGRESQL__SSLMODE`: Controls the SSL verification mode. Defaults to `verify-ca`.
+    - `disable`: No SSL is used.
+    - `allow`: Use SSL if available, but don't perform verification.
+    - `prefer`: Attempt an SSL connection first, fall back to non-SSL if it fails.
+    - `require`: Require an SSL connection, but without certificate verification.
+    - `verify-ca`: Require SSL and verify that the server certificate is signed by a trusted CA.
+    - `verify-full`: Require SSL, verify the CA, and verify that the server hostname matches the certificate.
+
+- `AUTHENTIK_POSTGRESQL__SSLROOTCERT`: Path to the CA certificate file for verifying the server's certificate. Required for `verify-ca` and `verify-full` modes.
+
+- `AUTHENTIK_POSTGRESQL__SSLCERT`: Path to the client SSL certificate file. Required if the PostgreSQL server is configured for mutual TLS and requires client certificates.
+
+- `AUTHENTIK_POSTGRESQL__SSLKEY`: Path to the private key for the client certificate (`SSLCERT`).
 
-    The name of the schema used by default in the database. Defaults to `public`.
+For more details, see [Django's PostgreSQL documentation](https://docs.djangoproject.com/en/stable/ref/databases/#postgresql-connection-settings) or the [PostgreSQL documentation](https://www.postgresql.org/docs/current/libpq-ssl.html#LIBPQ-SSL-PROTECTION).
 
-    It is only supported to change this value before the initial start of authentik.
+### Connection Management
 
-    When setting this value to any non-standard value, the schema must already exist in the database, and the user authentik connects with must have access to it. Additionally the `search_path` of the authentik database must be set to include this schema.
+These settings control connection persistence and behavior, which is particularly important when using a connection pooler like PgBouncer.
 
-### Read replicas
+- `AUTHENTIK_POSTGRESQL__CONN_MAX_AGE`: The maximum age of a database connection in seconds.
+    - `0` (default): Connections are closed after each request.
+    - `> 0`: Enables persistent connections, with the value defining the maximum lifetime.
```

Comment:
```suggestion
    - greater than `0`: Enables persistent connections, with the value defining the maximum lifetime.
```
Much more readable, and doesn't imply that `> 0` is a valid setting, which it isn't.